main/src/cgeo/geocaching/settings/SettingsActivity.java

```diff
@@ -757,6 +758,22 @@ private void bindSummaryToStringValue(final int key) {
         bindSummaryToValue(pref, value);
     }
 
+    private void bindGeocachingUserToGCVoteuser() {
+        final Preference pref = getPreference(R.string.pref_username);
+
+        if (pref == null) {
+            return;
+        }
+
+        final String value = PreferenceManager
```

Comment:
You should rather directly go the way to ```Settings.getGcCredentials()``` (perhaps first checking with ```Settings.hasGcCredentials()```) instead of this more complicated and harder to understand way through the preference.